tests/io/test_path.py

```diff
@@ -300,3 +301,6 @@ def test_backwards_compat(self):
 
         with pytest.raises(AttributeError):
             get_fs("file", storage_options={"foo": "bar"})
+
+        # Reset the cache to avoid side effects
+        _register_filesystems.cache_clear()
```

Comment:
Nice. Recalls the `recent cache clearing issue` :)

Reply:
But I think It would be better to use `try/finally` or - better - pytest fixture. This way the cache will be cleared even if the test fails @bolkedebruin

Reply:
see https://github.com/apache/airflow/pull/35746